bin/main.cpp

```diff
@@ -32,6 +32,8 @@ int main(int argc, char *argv[])
       else if (mode == "build_rom")    BuildROM(MPI_COMM_WORLD);
       else if (mode == "train_rom")    TrainROM(MPI_COMM_WORLD);
       else if (mode == "single_run")   double dump = SingleRun(MPI_COMM_WORLD, output_file);
+      else if (mode == "test")    TEMPRunAndCompare();
```

Comment:
I'd remove `TEMPRunAndCompare` and `OutputOperators`. They're mostly for testing/debugging purposes and don't suit the use for the main driver.

Reply:
Done now! Thanks, I totally forgot about these.